src/backend/distributed/sql/udfs/update_colocation/latest.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE FUNCTION pg_catalog.update_colocation(table_name regclass, colocate_with text)
```

Comment:
Still unsure about the naming. It's not very clear that this is a Citus function for distributed tables.

Maybe:
- update_distributed_table_colocation
- update_distributed_table (could be used for changing distribution column in the future)

Reply:
Chose update_distributed_table_colocation.